code/modules/materials/definitions/solids/_mat_solid.dm

```diff
@@ -4,6 +4,7 @@
 	boiling_point = 30000
 	door_icon_base = "stone"
 	icon_base = 'icons/turf/walls/stone.dmi'
+	wall_paintable = 0
```

Comment:
Redundant with the declaration on /decl/material being 0 already I think.

Reply:
You should decide on what the initial value should be and remove the redundant declarations after that.